README.md

```diff
@@ -1 +1,47 @@
-# Riyaz-Portfolio
\ No newline at end of file
+# Riyaz-Portfolio
+
+## Built With
+
+HTML
+CSS
+
+#### Getting Started
+
+https://r-ahmed2022.github.io/Riyaz-Portfolio/
+
+#### Getting Started
+
+Instructions on how to access my project locally. To Access my project locally please follow the following instructions;
+
+Enter this url: https://github.com/r-ahmed2022/Riyaz-Portfolio in your web browser.
+Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+Select download Zip option from drop down menu.
+
+## Prerequisites
+
+-> Setup GitHub repository
+-> Install git
+-> Setup Linters
+. Lighthouse
+. Weblint
+. Stylelint
+
+Deployment
+Authors
+👤 Riyaz
+
+GitHub: @https://github.com/r-ahmed2022/
+LinkedIn: https://www.linkedin.com/in/riyaz-ahmed
```

Comment:
[OPTIONAL] I recommend you link your social media using markup, it'll make it look more professional. All you need to do is wrap the text you want to link in square brackets([]) and then wrap the actual link you want to re-direct the user to in parenthesis (()). That has the [following effect](https://www.google.com/).